File: apps/s-2-admin/src/api/eventTypeCustomInput/EventTypeCustomInputCreateInput.ts
import { EventTypeWhereUniqueInput } from "../eventType/EventTypeWhereUniqueInput";

export type EventTypeCustomInputCreateInput = {
  eventType: EventTypeWhereUniqueInput;
  label: string;
  type: "TEXT" | "TEXTLONG" | "NUMBER" | "BOOL";
  required: boolean;
  placeholder: string;
};

File: apps/cactus-demo/src/APIs/EventTypeCustomInput/Dtos/EventTypeCustomInputCreateInput.cs
using CactusDemo.Core.Enums;

namespace CactusDemo.APIs.Dtos;

public class EventTypeCustomInputCreateInput
{
    public int? Id { get; set; }

    public string Label { get; set; }

    public TypeEnum Type { get; set; }

    public bool Required { get; set; }

    public string Placeholder { get; set; }

    public EventType EventType { get; set; }
}
